packages/components/forms/src/Select/Select.styles.ts

```diff
@@ -28,6 +43,7 @@ export function getSelectStyles({ isInvalid, isDisabled, size }) {
     outline: 'none',
     border: `1px solid ${tokens.gray300}`,
     cursor: 'pointer',
+    textOverflow: 'ellipsis',
```

Comment:
The `text-overflow` handling was missing, resulting in cut text when too long

| Before | After |
|--------|--------|
|  <img width="521" alt="no-ellipsis" src="https://github.com/contentful/forma-36/assets/103024358/3de39428-3d2d-4b49-a881-ff5b1a4a6fc5"> | <img width="524" alt="with-ellipsis" src="https://github.com/contentful/forma-36/assets/103024358/65a654b4-e174-42df-8d51-dc02da62b340"> |